ste/xfer-anyToRemote-file.go

```diff
@@ -181,6 +181,8 @@ func anyToRemote(jptm IJobPartTransferMgr, p pipeline.Pipeline, pacer pacer, sen
 
 	if info.IsFolderPropertiesTransfer() {
 		anyToRemote_folder(jptm, info, p, pacer, senderFactory, sipf)
+	} else if (jptm.GetOverwriteOption() == common.EOverwriteOption.PosixProperties() && info.EntityType == common.EEntityType.File()) {
```

Comment:
❓ What if a user specifies overwrite properties, but ends up uploading new files?

Reply:
The next line would fail if the file does not exist, it does not upload the file